src/api/spec/factories/patchinfo.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  factory :patchinfo, class: 'Patchinfo' do
+    # Patchinfo is an ActiveModel::Model. Override FactoryBot `create` method
+    to_create { nil }
```

Comment:
@eduardoj maybe you can use `skip_create` here? https://github.com/thoughtbot/factory_bot/blob/main/GETTING_STARTED.md#custom-methods-to-persist-objects

Reply:
@krauselukas yes. Thanks!